docs/wiki/archive/Magnetometer.mdx

```diff
@@ -17,56 +20,65 @@ Mag information is essential for position hold, which we intend to support.
 
 GPS Rescue does not require Mag data, but heading control may be improved if reliable, accurate Mag information is available, especially during descents on windy days. Note: DO NOT enable Mag if you rely on GPS rescue, unless you are ABSOLUTELY CERTAIN that the Mag data is accurate and reliable, and have confirmed that enabling Mag makes a significant improvement in rescue behaviour!
 
-## What is a Magenetometer?
+## What is a Magenetometer? {#magnetometer-explanation}
 
-A magnetometer is a three-axis device that detects the strength of the earth's magnetic field on each of three orthogonal sensor axes.
+A magnetometer is a three-axis device that detects the local strength and direction of the [earth's magnetic field](https://en.wikipedia.org/wiki/Earth%27s_magnetic_field)[^1].
 
-Magnetometers output data on three axes: X, Y and Z. Typically, these axes are arranged as follows:
+Magnetometers output data on three axes: X, Y and Z. The orientation of the axes can be arbitrary, depending on the magnetometer model. But typically, the axes are arranged as follows:
 
```

Comment:
All magnetometer sensor chip that we support have a consistent *relative relationship* between their three axes.  In nearly every case, the Z axis is upwards if the sensor is soldered to the top of the board, and downwards when soldered to the bottom of the board.  And, whether on the top, or the bottom, the sensor can be rotated X forward, or back, etc.

Maybe something like this:

```
Magnetometer sensors output data on three mutually perpendicular axes: X, Y and Z.  All sensors supported by Betaflight have the same relative relationship between the three axes: if Z is up, and X is forward, Y always points left.  

The sensor chip may be soldered on the top of the board, e.g. most standalone modules such as the GY-271.  Or, it can be soldered upside down, on the bottom of the board, like most GPS modules.  Additionally, the sensor can be soldered so that it's X axis faces forward, or 180 degrees backwards, or left, or right.  

If the magnetometer has arrows indicating the direction of the X and Y axes on the board, or if the user can determine the direction of the axes from the sensor's datasheet, the direction of the Z axis will be:
 - pointing upwards if the Y axis is 90° to the left of X axis
 - pointing downwards if the Y axis is 90° to the right of the X axis
```